docs/de/docs/index.md

```diff
@@ -107,16 +102,16 @@ The key features are:
 
 If you are building a <abbr title="Command Line Interface">CLI</abbr> app to be used in the terminal instead of a web API, check out <a href="https://typer.tiangolo.com/" class="external-link" target="_blank">**Typer**</a>.
 
-**Typer** is FastAPI's little sibling. And it's intended to be the **FastAPI of CLIs**. ⌨️ 🚀
+**Typer** ist das kleine Schwesterchen von FastAPI. Und es soll die **FastAPI der CLIs** sein. ⌨️ 🚀
 
-## Requirements
+## Anforderungen
 
 Python 3.6+
 
-FastAPI stands on the shoulders of giants:
+FastAPI basiert auf den Projekten:
 
-* <a href="https://www.starlette.io/" class="external-link" target="_blank">Starlette</a> for the web parts.
-* <a href="https://pydantic-docs.helpmanual.io/" class="external-link" target="_blank">Pydantic</a> for the data parts.
+* <a href="https://www.starlette.io/" class="external-link" target="_blank">Starlette</a> für die Webteile.
```

Comment:
Maybe `Webinhalte`and `Dateninhalte` instead of `-teile`

Reply:
Or: Webanteile, Datenanteile